tensorboard/backend/event_processing/data_ingester_test.py

```diff
@@ -134,38 +134,52 @@ def testPathWithColonThatComesAfterASlash_isNotConsideredARunName(self):
         )
 
     def testExpandsUser(self):
-        oldhome = os.environ.get("HOME", None)
+        old_home = os.environ.get("HOME", None)
+        old_userprofile = os.environ.get("USERPROFILE", None)
         try:
-            os.environ["HOME"] = "/usr/eliza"
+            test_home_directory = "/usr/eliza"
+            os.environ["HOME"] = test_home_directory
+            os.environ["USERPROFILE"] = test_home_directory
```

Comment:
Let's not set this for the posix one since USERPROFILE is only used for Windows.

Reply:
Done, and added a comment.